client/src/components/Comments/index.tsx

```diff
@@ -125,6 +133,39 @@ class Comments extends React.Component<ICommentsProps> {
             )
     )
 
+    public renderCommentsTemplate = (commentInfo: ICommentInfo): JSX.Element => {
+        const {authorId, comment}: ICommentInfo = commentInfo;
+        return(
+        <div className='one-comment px-3'>
+            <div className='d-flex justify-content-between'>
+                <div className='w-100'>
+                    <img
+                        src={this.props.authors[authorId].photoPath || noAvatar}
+                        alt='avatar'
+                        width={24}
+                        height={24}
+                        className='img-fluid rounded-circle mt-1 mr-1 mb-1'
+                    />
+                    <Link to={`/profile/${ this.props.authors[authorId].username}`}
+                          className='text-dark mt-1'
+                    >
+                        { this.props.authors[authorId].username}
+                    </Link>
+                    {
+                        (
+                            this.props.feed.loggedUsername ===  this.props.authors[authorId].username
```

Comment:
u use a lot of props. can u use destruction ?

Reply:
fixed